src/Controller/PromotionalCollectionsController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace eLife\Journal\Controller;
+
+use eLife\ApiSdk\Collection\Sequence;
+use eLife\ApiSdk\Model\Identifier;
+use eLife\ApiSdk\Model\PromotionalCollection;
+use eLife\Journal\Helper\Callback;
+use eLife\Patterns\ViewModel\ContentHeader;
+use eLife\Patterns\ViewModel\ContextualData;
+use eLife\Patterns\ViewModel\ListHeading;
+use eLife\Patterns\ViewModel\ListingProfileSnippets;
+use eLife\Patterns\ViewModel\ListingTeasers;
+use eLife\Patterns\ViewModel\ProfileSnippet;
+use eLife\Patterns\ViewModel\Teaser;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+final class PromotionalCollectionsController extends Controller
+{
+    public function promotionalCollectionAction(Request $request, string $id) : Response
+    {
+        $arguments['item'] = $this->get('elife.api_sdk.promotional-collections')
+            ->get($id)
+            ->otherwise($this->mightNotExist())
+            ->then($this->checkSlug($request, Callback::method('getTitle')));
+
+        $arguments = $this->defaultPageArguments($request, $arguments['item']);
+
+        $arguments['title'] = $arguments['item']
+            ->then(Callback::method('getTitle'));
+
+        $arguments['pageViews'] = $this->get('elife.api_sdk.metrics')
+            ->totalPageViews(Identifier::promotionalCollection($id))
```

Comment:
is this already in `elife-metrics` or does it need to be explicitly configured? Sounds like a soft failure will always be there for new pages.

Reply:
> is this already in `elife-metrics` or does it need to be explicitly configured? Sounds like a soft failure will always be there for new pages.

This needs some attention.

Reply:
Quick search leads me to https://github.com/elifesciences/elife-metrics/blob/b9fdbb03c3c06366bad5e0f35e458cac84ed745f/schema/metrics/history.json, we need a ticket for @lsh-0